### ISSUE TITLE
Add specificities for 1 column data

### ISSUE THREAD

**Issue by dragoudin** [jenkinsci/nested-data-reporting-plugin#33]
### What feature do you want to see added?

In the case of 1 column data only, it would be good to have for Overview / History area consistent data (because if one column, it will be full green...).
In this case, in overview and history we would like to see the first level of data:

      Atkie1
      Atkie2
      Atkie3...

Usually, my use of this plugin is for 1 column data (and I will create multiple report for each column). I want to see the repartition of data for each column parameter

**Comment by simonsymhoven** [comment]
Hi @dragoudin, thanks for you request. Please can you provide more details? I am not sure if I understand what you exactly mean.

**Comment by dragoudin** [comment]
Hi
I consider the example provided. If we consider that we have only one column (for example "incorrect" only).
In the overview, we will get only "incorrect" (100% of my data).
So in case of one column, instead having statistics for each column, it would be better to have statistics on top level rows (Atkie1...). These data would be more suggestive.

In case of multiple columns, then for overview, viewing each proportions of columns make sense.

![image](https://user-images.githubusercontent.com/94975153/188477532-92ddecdf-9dd1-4c78-a43f-cdf75e6cf294.png)

**Comment by simonsymhoven** [comment]
@dragoudin 
I implemented this request, can you please have a look at it:

![Bildschirmfoto 2022-09-06 um 09 55 55](https://user-images.githubusercontent.com/26878018/188579081-3d330649-41c3-4cc0-9ce6-aff5a7ab3ed3.png)

The color mapping will be quite long, because you need a color for each item, e.g.:

```
{
  "items": [
    {
      "id": "Aktie",
      "name": "Aktie",
      "items": [
        {
          "id": "Aktie_1",
          "name": "Aktie 1",
          "result": {
            "incorrect": 3541
          }
        },
        {
          "id": "Aktie_2",
          "name": "Aktie 2",
          "result": {
            "incorrect": 4488
          }
        },
        {
          "id": "Aktie_3",
          "name": "Aktie 3",
          "result": {
            "incorrect": 2973
          }
        }
      ]
    },
    {
      "id": "Not_Found",
      "name": "Not_Found",
      "result": {
        "incorrect": 8701
      }
    },  
    {
      "id": "Renten",
      "name": "Renten",
      "items": [
        {
          "id": "Rente_1",
          "name": "Rente1",
          "result": {
            "incorrect": 5762
          }
        },
        {
          "id": "Rente_2",
          "name": "Rente2",
          "result": {
            "incorrect": 2271
          }
        }
      ]
    },
    {
      "id": "Derivat",
      "name": "Derivat",
      "result": {
        "incorrect": 2271
      }
    }
  ],
  "colors": {
    "Aktie": "#EF9A9A",
    "Aktie_1": "#FFF59D",
    "Aktie_2": "#A5D6A7",
    "Aktie_3": "#FFCE30",
    "Not_Found": "#E83845",
    "Renten": "#E389B9",
    "Rente_1": "#746AB0",
    "Rente_2": "#288BA8",
    "Derivat": "#0088FF"
  }
}
```

**Comment by simonsymhoven** [comment]
closed by #37
